src/frontend/Felles/Knapper/DisplayBlockKnapp.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styled from 'styled-components';
+import FjernKnapp from './FjernKnapp';
+import LeggTilKnapp from './LeggTilKnapp';
+import HovedKnapp from './HovedKnapp';
+
+const Div = styled.div`
+    display: block;
+`;
+
+interface Props {
+    className?: string;
+    disabled?: boolean;
+    ikontekst?: string;
+    knappetekst?: string;
+    onClick?: () => void;
```

Comment:
Samme som lenger ned, hvorfor er onclick valgfri på en knapp?

Reply:
Dersom knappen er av typen `submit` eller `reset` behøver den ingen onClick for å fyre av en funksjon